package/src/kustoMode.ts

```diff
@@ -8,6 +8,7 @@ import type { IKustoWorkerImpl } from './kustoWorker';
 import { kustoLanguageDefinition } from './syntaxHighlighting/kustoMonarchLanguageDefinition';
 import { LANGUAGE_ID } from './globals';
 import { semanticTokensProviderRegistrarCreator } from './syntaxHighlighting/semanticTokensProviderRegistrar';
+import { IDisposable } from 'monaco-editor';
```

Comment:
@morgilad I'm pretty certain this is going to cause bundle size issues. We need to be extremely careful where and how we import `monaco-editor`.

If we only want the type we can prefix it with `type`:

`import type { IDisposable } from 'monaco-editor';

Reply:
If it _doesn't_ cause bundle size issues, then we've likely added another bad import somehwere else.